MapboxCoreNavigationTests/CocoaPodsTest/PodInstall/Podfile.lock

```diff
@@ -13,9 +13,9 @@ PODS:
     - Polyline (~> 4.2)
     - Turf (~> 0.5.0)
   - MapboxMobileEvents (0.10.2)
-  - MapboxNavigation (1.1.0-alpha.1):
-    - Mapbox-iOS-SDK (= 6.1.0-beta.2)
-    - MapboxCoreNavigation (= 1.1.0-alpha.1)
+  - MapboxNavigation (1.0.0-rc.3):
+    - Mapbox-iOS-SDK (~> 6.0)
+    - MapboxCoreNavigation (= 1.0.0-rc.3)
```

Comment:
For the master branch, this needs to be v1.1.0-alpha.1 or so.

Reply:
This will bring inconsistency to other places (since `v1.0.0-rc.3` was released later than `v1.1.0-alpha.1` `README.md` references `v1.0.0-rc.3` as well). I think it's better to keep using `v1.0.0-rc.3` in `master` and update it to `v1.1.0-alpha.2` whenever we're ready to do so.